template.html

```diff
@@ -68,19 +70,19 @@ <h2><a href="#users">
               <button class="details-toggle" data-toggle-target="users"><span class="sr-only">Details in-/uitklappen...</span></button>
             </section>
             <section class="details">
-              <p>{{topic.users.how-title}}</p>
+              <h3>{{topic.users.how-title}}</h3>
```

Comment:
Great change for A11Y of course, thx! One suggestion though: could we tweak the css for `h3` a little so the design doesn't change _as much_ from the original? That is, way.infi.nl currently has a much smaller font for these bits of text.

No need to be pixel-perfect, but the deploy preview now makes the header far too close to the `h2` look and feel (thus hurting accessibility I reckon, because the header levels become less distinguishable by users with varying sight levels).